conf/bookkeeper.conf

```diff
@@ -631,10 +631,10 @@ diskCheckInterval=10000
 ## Metadata Service settings
 #############################################################################
 
-# metadata service uri that bookkeeper is used for loading corresponding metadata driver and resolving its metadata service location
+# Metadata service uri that bookkeeper uses for loading the corresponding metadata driver and resolving its metadata service location
 # Examples: 
-#  - metadataServiceUri=metadata-store:zk:my-zk-1:2181
-#  - metadataServiceUri=metadata-store:etcd:http://my-etcd:2379
+#  - metadataServiceUri=zk://my-zk-1:2181/ledgers
+#  - metadataServiceUri=etcd:http://my-etcd:2379
```

Comment:
FYI @momo-jun @daveduggins

Reply:
> Hi @DaveDuggins Does this look ok

Yes. Anonymitaet pinged me for reference. I'm not the approver. We still need a committer to merge this.